packages/raystack/rollup.config.mjs

```diff
@@ -5,49 +5,82 @@ import typescript from "@rollup/plugin-typescript";
 import postcss from "rollup-plugin-postcss";
 import postcssImport from 'postcss-import';
 
-const defaultGlobals = {
-  react: "React",
-  "react-dom": "ReactDOM",
-};
-
-export default {
-  input: "index.tsx",
-  output: [
-    {
-      dir: "dist",
-      format: "es",
-      sourcemap: true,
-      exports: "named",
+const createPlugins = () => [
+  nodeResolve(),
+  commonjs(),
+  postcss({
+    plugins: [
+      postcssImport(),
+    ],
+    extract: 'style.css',
```

Comment:
Kept the name same as previous styles file to keep the style file import same as before.